src/scripts/main.js

```diff
@@ -1,3 +1,13 @@
 'use strict';
 
-// write code here
+function sortList(list) {
+  let myList = list;
+
+  myList = document.querySelector('ul');
+
+  const salaryElement = myList.querySelector('data-salary');
```

Comment:
The `querySelector` method is used incorrectly. To select an element by a custom data attribute, you should use `myList.querySelector('[data-salary]')` instead.